backend/Persistence/DB/Data/Test/Foods.json

```diff
@@ -1,150 +1,218 @@
 [
   {
-    "Price": 9.25,
-    "CreatedAt": "2008-06-12T00:00:00",
+    "StartingPrice": 15,
+    "CreatedAt": "2021-11-04T11:47:04.9955968+02:00",
+    "IdRestaurant": "0",
+    "TypesOfFood": [],
+    "StartDecreasingAt": "2021-11-04T11:47:04.9955968+02:00",
+    "DecreaseType": "PERCENT",
```

Comment:
I think we should use enums for `DecreaseType`, for example 0 for percent type, and 1 for value type